MAINTENANCE.md

```diff
@@ -0,0 +1,80 @@
+# Roles, People, and Processes for Maintaining the Tagging Schema
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project as part of his work on the iD Editor project.
+
+The maintainer role includes:
+- Having the final say in decisions for the tagging schema.
+- Creating releases.
+- Updating dependencies.
+
+and all the following roles.
+
+### Contributor Role
```

Comment:
Maybe different name may be better? Github is using "contributor" to refer to anyone who got commit accepted and to anyone who opened a PR.

![screen10](https://github.com/openstreetmap/id-tagging-schema/assets/899988/32c000e8-f2b4-44bb-95c7-95db6ebc5f90)

Maybe "Helper"?

Reply:
Good point. Github also uses "collaborator" …
> <img width="537" alt="image" src="https://github.com/openstreetmap/id-tagging-schema/assets/111561/eb80dbfb-c6e2-47a3-a1db-2772c34a557e">

I think that badge appeared once I had triage roles but I am not sure if the name changed. I remember that the triage role added a name that suggested to readers that I had more of a maintainer role, which is not true.

Unfortunately https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization does not explain which badge is added when.

I think to clarify we should add the github badge name to this section so there is a reference.

We might need to test this out by changing my permission @tyrasd

Reply:
I added the names of the github labels as I understand them ATM. We will need to look for better names of the chapters/roles.

Reply:
"Collaborator" is when your account is added to project and has any rights assignable on github (at least from my understanding/memory).

"First-time contributor" is on PR of someone who has their first PR in repo (not sure is it overriding other labels)

"Contributor" - their changes were accepted into repo (not sure about edge cases like their PR merged that contained no commits they made or commits authored/committed by them being accepted without opening PR etc)

Reply:
I changed this role to _"Co-Maintainers"_. I think that better represents the huge added value by these volunteers.